src/main/java/com/getcloudcherry/survey/helper/RecordAnswer.java

```diff
@@ -126,14 +147,16 @@ public SurveyAnswers getAnswers() {
     public boolean checkIfPreFillExists(SurveyQuestions iQuestion) {
 
         for (String aTag : iQuestion.questionTags) {
```

Comment:
If the user hasn't entered any text in MuitiLine Textbox such as empty string we don't have to submit that response for both partial submit and normal submit (Ignore adding it to the responses array).